### DISCUSSION TITLE
Projects (Beta) - Add comments for draft issues

### DISCUSSION THREAD

**Discussion by yani-** [community/community#11361] [Projects and Issues]
Draft issues are lacking comments.
Comments need to be open for anyone with access to the Project.
Threaded comments will be even better as I can see the need to reply to a comment.

**Comment by Sulaiman739** [comment]
https://github.com/github/feedback/discussions/11361#discussion-3870237

**Comment by Sulaiman739** [comment]
https://github.com/github/feedback/discussions/11361#discussion-3870237
